pkg/gitutils/manifests.go

```diff
@@ -51,3 +75,102 @@ func CopyManifests(wt *gogit.Worktree, fs embed.FS, root string, mgmtPath string
 }
 
 // -----------------------------------------------------------------------------
+
+func CopyPatchManifests(wt *gogit.Worktree, filePath string, clusterPath string,
+	baseRepoUrl string, baseRepoPath string, baseRepoRevision string) error {
+	log := log.GetLogger()
+	files, err := ioutil.ReadDir(filePath)
+	if err != nil {
+		fmt.Println(err)
+		return fmt.Errorf("Failed to read the directory %s", err)
+	}
+	var targetData []target
+	for _, file := range files {
+		newFilePath := path.Join(filePath, file.Name())
+		src, err := os.OpenFile(newFilePath, os.O_RDONLY, os.ModePerm)
+		if err != nil {
+			_ = src.Close()
+			return fmt.Errorf("failed to open embedded file %s: %s", filePath, err)
```

Comment:
Why does this say `embedded`? Should it say `patch file` instead?

Reply:
renamed it to patch file